internal/config/config.go

```diff
@@ -57,6 +58,9 @@ type Config struct {
 
 	// OIDC client ID for application
 	ClientID string
+	// File containing the OIDC Client Secret
+	clientSecretFile string
```

Comment:
```suggestion
	ClientSecretFile string
```

Reply:
@cpanato Emulating the design in [cosign](https://github.com/sigstore/cosign/blob/main/cmd/cosign/cli/options/oidc.go#L33).

Testing issues resolved and checks are ✅